### ISSUE TITLE
error "wait init..."

### ISSUE THREAD

**Issue by koronat** [slightc/pip-manager#18]
I`ve read previous reports of the issue. I`ve tried all the solution proposed.
Have this problem with different versions of pip-manager (tried version 1.0-1.1).
At first I did not even see that the manager started. I overcame this and faced another problem.
Yes, I do run Python. I use Mac OS 11.5.1.
This feature is super helpful, so I hope this issue can be resolved somehow.

Here`s what I got.
<img width="1327" alt="Screenshot 2022-10-13 at 15 16 06" src="https://user-images.githubusercontent.com/115711769/195595015-20510929-5de5-423b-9978-c0bd8a06cf63.png">
<img width="1023" alt="Screenshot 2022-10-13 at 15 16 14" src="https://user-images.githubusercontent.com/115711769/195595037-2f2ab90f-a5b7-4268-a70a-39104560545b.png">

**Comment by slightc** [comment]
This maybe pip error, you can try reinstall you pip

Hope this issue can help you
https://github.com/pypa/pip/issues/8263

**Comment by koronat** [comment]
Lovely, thank you. 
It helped me partially.

I tried to reinstall pip, VSC and python, but it did not work. 
Finally, I got tired and deleted everything (pip, python, VSC) and cleaned remaining files in /usr/ location and /library/ as well (as was suggested in mentioned thread). I reloaded my Mac and installed everything again causiously. 
It worked only after installed pip 1.1.0 (current version 1.1.1 did not work). 
Hope that would help to someone.

Thanks slightc!
